cli/commands/check/interactive.go

```diff
@@ -236,13 +236,18 @@ func (opts *checkOpts) administerQuestionnaire(editing bool) error {
 		},
 		{
 			Name: "output-metric-format",
-			Prompt: &survey.Input{
+			Prompt: &survey.Select{
 				Message: "Metric Format:",
-				Help:    "Optional output metric format used to parse check output for metric extraction. Valid formats include: nagios_perfdata, graphite_plaintext, opentsdb_line, and influxdb_line",
-				Default: opts.OutputMetricFormat,
+				Options: []string{
+					"nagios_perfdata",
```

Comment:
We have some constants for these formats, see https://github.com/sensu/sensu-go/blob/6996d8ccb36ba9a9b6750da016ce3b5ea118bcfd/types/check.go#L40

Reply:
Oops, I merged before I noticed that. :man_facepalming: